openresty/Dockerfile-alpine

```diff
@@ -232,7 +233,9 @@ RUN set -eux; \
         sed \
         yajl; \
     # Install lua modules here
-    luarocks install lua-resty-openidc; \
+    for module in lua-resty-openidc ${LUA_MODULES}; do \
```

Comment:
Now I think it makes sense to move `lua-resty-openidc` to the bake file as well.